data/result-exporter/cne-exporter-commons/src/main/java/com/powsybl/openrao/data/cneexportercommons/CneHelper.java

```diff
@@ -8,67 +8,119 @@
 package com.powsybl.openrao.data.cneexportercommons;
 
 import com.powsybl.openrao.data.cracapi.Crac;
-import com.powsybl.openrao.data.cracapi.Instant;
 import com.powsybl.openrao.data.raoresultapi.RaoResult;
-import com.powsybl.openrao.raoapi.parameters.RaoParameters;
-import com.powsybl.openrao.raoapi.parameters.extensions.LoopFlowParametersExtension;
-import com.powsybl.openrao.raoapi.parameters.extensions.MnecParametersExtension;
 
-import static com.powsybl.openrao.data.cneexportercommons.CneConstants.PATL_MEASUREMENT_TYPE;
-import static com.powsybl.openrao.data.cneexportercommons.CneConstants.TATL_MEASUREMENT_TYPE;
+import java.util.Properties;
 
 /**
  * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
  * @author Peter Mitri {@literal <peter.mitri at rte-france.com>}
  */
 public class CneHelper {
+    private final Crac crac;
+    private final RaoResult raoResult;
+    private final boolean relativePositiveMargins;
+    private final boolean withLoopFlows;
+    private final double mnecAcceptableMarginDiminution;
+    private final String documentId;
+    private final int revisionNumber;
+    private final String domainId;
+    private final String processType;
+    private final String senderId;
+    private final String senderRole;
+    private final String receiverId;
+    private final String receiverRole;
+    private final String timeInterval;
```

Comment:
no need to create all these attributes, just store the Properties object and use it when needed. put the property names as static final attributes in the CneConstants class